src/compatTest/kotlin/io/github/gradlenexus/publishplugin/NexusPublishPluginTests.kt

```diff
@@ -949,10 +949,19 @@ class NexusPublishPluginTests {
         gradleRunner(*arguments).buildAndFail()
 
     private fun gradleRunner(vararg arguments: String): GradleRunner {
+        val warnings = when {
+            // There's a deprecation in Gradle 7.6 that doesn't allow for passing builds.
+            // See https://github.com/gradle-nexus/publish-plugin/issues/152 for more details.
+            // This conditional branch of when should be removed once that issue is fixed.
+            gradleVersion >= GradleVersion.version("7.6") -> "all"
```

Comment:
```suggestion
            gradleVersion >= GradleVersion.version("7.6") -> "fail"
```
would fail, because of the warning in https://github.com/gradle-nexus/publish-plugin/issues/152

Note: should there be a scheduled CI job, that runs every day/week which runs the commands in OP and opens a PR. This would've helped discover https://github.com/gradle-nexus/publish-plugin/issues/152 by automation.

Reply:
Reverted in https://github.com/gradle-nexus/publish-plugin/pull/172/commits/3a607aa7496ea726e6edc3509d1fd396dc14ad98